Examples/Apply_scipy_function_to_every_gridpoint.ipynb

```diff
@@ -0,0 +1,533 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    plt.contourf(p_value.xt_ocean,p_value.yt_ocean,p_value,levels=(0,0.05),colors='None',hatches=('...',))
</pre><p>can we have spaces after commas? that's a general comment but it's quite prominent here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/421/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>